worms_game.py

```diff
@@ -108,15 +129,18 @@ def _handle_events(self):
 
             # Mouse events
             if event.type == pygame.MOUSEBUTTONDOWN:
-                if not self.current_projectile:
-                    self.current_projectile = Projectile(
-                        self.current_worm.rect.center, pygame.mouse.get_pos()
-                    )
-                    self.current_projectile.start_charging()
-            if event.type == pygame.MOUSEBUTTONUP and self.current_projectile:
-                self.current_projectile.stop_charging()
-                self.projectiles.add(self.current_projectile)
-                self.current_projectile = None
+                if event.button == 1:
```

Comment:
Ça c'est pour gérer le clic gauche ?

Si oui j'aurais fait un mini truc pour que ce soit plus compréhensible, genre un commentaire ou carrément créé une constante `LEFT_CLICK = 1` dans `globals.py`

Reply:
`            if event.type == pygame.MOUSEBUTTONDOWN:
                if event.button == g.MOUSE_LEFT_CLICK:
                    if not self.current_projectile:
                        self.current_projectile = Projectile(
                            self.current_worm.rect.center, pygame.mouse.get_pos()
                        )
                        self.current_projectile.start_charging()

            if event.type == pygame.MOUSEBUTTONUP:
                if event.button == g.MOUSE_LEFT_CLICK and self.current_projectile:
                    self.current_projectile.stop_charging()
                    self.projectiles.add(self.current_projectile)
                    self.current_projectile = None`
                    
                    <.<

Reply:
COMMENT TU FAIT POUR TAG UN CHANGE

Reply:
tu peux juste resolve le commentaire je saurai que t'as fait les modifs x)